lldb/source/Commands/CommandObjectProcess.cpp

```diff
@@ -1591,24 +1591,24 @@ class CommandObjectProcessHandle : public CommandObjectParsed {
 
   Options *GetOptions() override { return &m_options; }
 
-  bool VerifyCommandOptionValue(const std::string &option, int &real_value) {
-    bool okay = true;
+  std::optional<bool> VerifyCommandOptionValue(const std::string &option) {
```

Comment:
We should modify `OptionArgParser::ToBoolean(...)` to return a `std::optional<bool>`. it already handles "0" and "1" strings correctly, so this integer conversion stuff is just not doing anything useful

Reply:
I was actually in the middle of doing that when I found `VerifyCommandOptionValue`. I was going to upload a PR for it after removing this.